baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -51,13 +53,27 @@ class AmqpConsumerPrometheusLabels(NamedTuple):
     AmqpConsumerPrometheusLabels._fields + ("amqp_success",),
 )
 
+AMQP_REPUBLISHED_TOTAL = Counter(
```

Comment:
are there associated spec changes for these new metrics?

Reply:
Sorry, I don't understand; which spec are we talking about? Any reference or example to look at, please?

Reply:
Ohh,  you probably meant tests? Sure, will do; I only managed to run it in snoodev today, the tests are next.

Reply:
Added unit tests.